BookingsApi/BookingsApi.Infrastructure.Services/IntegrationEvents/Events/AllocationHearingsIntegrationEvent.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using BookingsApi.Domain;
+using BookingsApi.Domain.Participants;
+using BookingsApi.Infrastructure.Services.Dtos;
+
+namespace BookingsApi.Infrastructure.Services.IntegrationEvents.Events
+{
+    public class AllocationHearingsIntegrationEvent: IIntegrationEvent
+    {
+        public AllocationHearingsIntegrationEvent(List<VideoHearing> hearings, JusticeUser allocatedCso)
+        {
+            Hearings = hearings.Select(HearingDtoMapper.MapToDto).ToList();
+            AllocatedCso = (allocatedCso != null) ? UserDtoMapper.MapToDto(allocatedCso) : null;
+        }
+
+        public List<HearingAllocationDto> Hearings { get; }
```

Comment:
I probably was not clear enough. What I meant was something like below:
```suggestion
public List<HearingDto> Hearings { get; }
public UserDto AllocatedCso { get; }
public ParticipantDto Judge { get; }
```

The judge does not belong on the hearing dto because it has no relation to the hearing's details.

Reply:
created a new dto

Reply:
I am not sure how this will work. We have a list of hearings. each hearing has a different judge

Reply:
It would work just the same way as you pull the display name. you find the judge through the list of participants, and if one is present then map it

Reply:
I don't think you understand the model here. We have a list of hearings. Each hearing has one judge. In your suggestion you have only one judge. How this will work?